typescript/infra/config/environments/testnet/agent.ts

```diff
@@ -28,7 +28,11 @@ export const abacus: AgentConfig<TestnetChains> = {
     chainOverrides: {
       optimismkovan: {
         interval: 5,
-        reorgPeriod: 2,
+        reorgPeriod: 0,
+      },
+      arbitrumrinkeby: {
+        interval: 5,
+        reorgPeriod: 0,
```

Comment:
if the `reorgPeriod` wasn't here, would it still be `0` or are the configs not "deep" merged?